components/region_cache_memory_engine/src/engine.rs

```diff
@@ -24,11 +23,12 @@ use crate::keys::{
     VALUE_TYPE_FOR_SEEK, VALUE_TYPE_FOR_SEEK_FOR_PREV,
 };
 
-fn cf_to_id(cf: &str) -> usize {
+pub(crate) fn cf_to_id(cf: &str) -> usize {
     match cf {
         CF_DEFAULT => 0,
         CF_LOCK => 1,
         CF_WRITE => 2,
+        CF_RAFT => 3,
```

Comment:
CF_RAFT should not be included. The engine here is KvEngine, which shoud only includeds the above three CFs.

Reply:
Yes, that was my expectation, but it ended up causing failures. Something needs to be fixed in the code: right now, it complains about missing CF_RAFT when initialized in component/server/src/server.rs.